examples/asr/transcribe_speech.py

```diff
@@ -151,10 +232,22 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
     else:
         # get filenames from manifest
         filepaths = []
+        if os.stat(cfg.dataset_manifest).st_size == 0:
+            logging.error(f"The input dataset_manifest {cfg.dataset_manifest} is empty. Exiting!")
+            return None
+
         with open(cfg.dataset_manifest, 'r') as f:
+            partial_audio = True
             for line in f:
                 item = json.loads(line)
+                if "duration" not in item:
+                    partial_audio = False
                 filepaths.append(item['audio_filepath'])
+        if not partial_audio:
```

Comment:
Do not raise this warning - by default most manifest do not have offset set. We don't want to warm for a general case